_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,147 @@
+- [A working definition of Government as a Platform](https://medium.com/digitalhks/a-working-definition-of-government-as-a-platform-1fa6ff2f8e8d){:target="_blank"}, is an essay from Richard Pope presenting how governments can become platforms from which citizen services can be designed by reusing components and services.
+- [Lessons from Estonia on digital government](https://policyoptions.irpp.org/fr/magazines/february-2019/lessons-estonia-digital-government/){:target="_blank"} is an essay from David Eaves on how Canada could learn from Estonia's journey by leveraging common building components for digital services, even if we consider the complexity tied to the legacy systems and policies that Canada has to deal with.
+- [Web Experience Toolkit](https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/web-experience-toolkit.html){:target="_blank"} is a Government of Canada initiatives that "includes reusable components for building and maintaining innovative websites that are accessible, usable, and interoperable.
+  These reusable components are open source software and free for use by departments and external Web communities."
+  It is a great example of how a team was able to work in the open to benefit the government as a whole but also the society itself.
+- [GC Service & Digital Target Enterprise Architecture](https://wiki.gccollab.ca/index.php?title=GC_Service_%26_Digital_Target_Enterprise_Architecture&mobileaction=toggle_view_desktop){:target="_target"}
+- [1988 National Bureau of Standards' Management Guide to
+Software Reuse](https://www.govinfo.gov/content/pkg/GOVPUB-C13-dca04f7b7fc21321e39de88b35b2f1d3/pdf/GOVPUB-C13-dca04f7b7fc21321e39de88b35b2f1d3.pdf) (PDF)
+
+Needless to say that public administrations in general are ripe for reusing solutions and ideas to increase the quality of their services to citizens.
+
+### Manage secrets
+
+In software development, certain parts of the code may be a corporate secret, such as a private key to a resource, passwords, etc.
+
+These must never be included in the public source code itself but rather be managed separately.
+
+Since, software source code is nothing more than structured text that can be understood by a computer, the same concept of pulling out secrets parts may be taken with other types of documents.
+
+This means that when a small part of your work must not be shared publicly, you can exclude it and store it in a secure location instead of locking down the entire document.
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately.
+
+By default, you should strive to keep as many of your working documents in the open section of your workspace.
+But when needed, you can save sensitive information in the private folders and keep working in the open for the rest.
```

Comment:
I feel like rows 101 and 102 could be removed as you've already said this above.

Reply:
I believe this may actually be an oversight when restructuring the document 😅